Source/DafnyLanguageServer.Test/Various/CounterExampleTest.cs

```diff
@@ -84,25 +86,618 @@ method Abs(x: int) returns (y: int)
     [TestMethod]
     public async Task GetCounterExampleWithMultipleMethodsWithErrorsReturnsCounterExamplesForEveryMethod() {
       var source = @"
-method Abs(x: int) returns (y: int)
-    ensures y > 0
-{
-}
+      method Abs(x: int) returns (y: int)
+          ensures y > 0
+      {
+      }
 
-method Negate(a: int) returns (b: int)
-    ensures b == -a
-{
-}
-".TrimStart();
+      method Negate(a: int) returns (b: int)
+          ensures b == -a
+      {
+      }
+      ".TrimStart();
+      var documentItem = CreateTestDocument(source);
+      _client.OpenDocument(documentItem);
+      var counterExamples = (await RequestCounterExamples(documentItem.Uri)).ToArray();
+      Assert.AreEqual(2, counterExamples.Length);
+      Assert.AreEqual((2, 6), counterExamples[0].Position);
+      Assert.IsTrue(counterExamples[0].Variables.ContainsKey("y:int"));
+      Assert.AreEqual((7, 6), counterExamples[1].Position);
+      Assert.IsTrue(counterExamples[1].Variables.ContainsKey("a:int"));
+      Assert.IsTrue(counterExamples[1].Variables.ContainsKey("b:int"));
+    }
+
+    [TestMethod]
+    public async Task RealFull() {
```

Comment:
I went through the test method names and changed them whenever they were to cryptic. This particular method should have been (and now is) named `WholeNumberAsReal` since it tests the case when a whole number is represented by a value of type `real`.